src/scripts/main.js

```diff
@@ -1,3 +1,67 @@
 'use strict';
 
-// write code here
+const url = 'https://mate-academy.github.io/'
+  + 'phone-catalogue-static/api/phones';
+
+fetchData();
+
+function fetchData() {
+  fetch(`${url}.json`)
```

Comment:
```suggestion
  fetch(`${BASE_URL}/${url}`)
```
і я б писав `.json` в параметрах функції fetchData('phones.json')